src/pages/admin/donate/components/generic-details/GenericDetails.test.tsx

```
@@ -0,0 +1,379 @@
+        render(<GenericDetails {...defaultProps} initialIsItemsExpanded={false} />);
+
+        expect(screen.queryByText('Item 1')).not.toBeInTheDocument();
+
+        const header = screen.getByText('Test Title');
+        fireEvent.click(header);
+
+        expect(screen.getByText('Item 1')).toBeInTheDocument();
+    });
+
+    it('does not render children when children prop is undefined', () => {
+        render(<GenericDetails {...defaultProps} children={undefined} />);
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)